docs/en/changelog.md

```diff
@@ -1,5 +1,51 @@
 ## Changelog
 
+### V0.23.0 (4/1/2022)
+
+**Highlights**
+
+- Support BEiT: BERT Pre-Training of Image Transformers
+- Support K-Net: Towards Unified Image Segmentation
+- Add `avg_non_ignore` of CELoss to support average loss over non-ignored elements
+- Support dataset initialization with file client
+
+**New Features**
+
+- Support BEiT: BERT Pre-Training of Image Transformers ([1404](https://github.com/open-mmlab/mmsegmentation/pull/1404))
+- Support K-Net: Towards Unified Image Segmentation ([1289](https://github.com/open-mmlab/mmsegmentation/pull/1289))
+- Support dataset initialization with file client ([1402](https://github.com/open-mmlab/mmsegmentation/pull/1402))
+- Add class name function support for STARE datasets ([1376](https://github.com/open-mmlab/mmsegmentation/pull/1376))
+- Support different seeds to different ranks when distributed training ([1362](https://github.com/open-mmlab/mmsegmentation/pull/1362))
+- Add nlc2nchw2nlc and nchw2nlc2nchw ([1249](https://github.com/open-mmlab/mmsegmentation/pull/1249))
+
+**Improvements**
+
+- Synchronize random seed for distributed sampler ([#1411](https://github.com/open-mmlab/mmsegmentation/pull/1411))
+- Add faq document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```

Comment:
```suggestion
- Add FAQ document ([1420](https://github.com/open-mmlab/mmsegmentation/pull/1420))
```
FAQ is abbreviation for "frequently asked questions", which should use capital letter.